app/Http/Controllers/Marketing/Company/Handbook/HandbookController.php

```
@@ -0,0 +1,17 @@
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Marketing\Company\Handbook;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+final class HandbookController extends Controller
+{
+    public function index(Request $request): View
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)